torch/_refs/__init__.py

```diff
@@ -842,6 +881,7 @@ def float_power(
 # Sub is implemented specially because it has an alpha argument and
 #   is decomposed into multiple prims.
 # TODO: consider refactoring this with add impl
+@register_decomposition(torch.ops.aten.sub)
```

Comment:
We should add a note for decompositions/aten registration and add the pattern to the runbook, too